lib/zewo.rb

```diff
@@ -252,7 +226,7 @@ def uncommited_changes?(repo_name)
       end
 
       def master_branch?(repo_name)
-        name = `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
+        `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
       end
```

Comment:
this `name` variable isn't an instance variable right? that's why you removed it, correct?